addon/templates/components/sl-date-picker.hbs

```diff
@@ -1,11 +1,10 @@
 {{#if label}}
-    <label for={{sl-date-picker-view.elementId}}>{{label}}</label>
+    <label for={{inputElementId}}>{{label}}</label>
 {{/if}}
 
 {{input
     viewName="sl-date-picker-view"
```

Comment:
If getting rid of this property in the `label` element than can get rid of its assignment here.